src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/QualityGuidelines/AssigningSymbolAndItsMemberInSameStatementTests.cs

```diff
@@ -89,18 +161,40 @@ public class Test
     public void Method()
     {
         C a = new C(), b = new C();
-        a.Property.Property = a.Property = a = b;
+        [|a.Property.Property|] = [|a.Property|] = a = b;
```

Comment:
Not sure what should be the codefix(es) for this one too.

Reply:
Based upon my previous comment, I would say:

```csharp
a = b;
a.Property = a;
a.Property.Property = b;
```

and/or

```csharp
a = b;
a.Property = a;
a.Property.Property = a.Property;
```